src/todos/todos.service.spec.ts

```diff
@@ -0,0 +1,64 @@
+import { Test, TestingModule } from '@nestjs/testing'
+import { TodosService } from './todos.service'
+import { TodosDatabaseService } from './todos.database.service'
+import type { Todo } from './schemas/todos.schema'
+import { createMock } from '@golevelup/ts-jest'
+
+describe('TodosService', () => {
+  let todosService: TodosService
+  let todosDatabaseService: TodosDatabaseService
+
+  beforeEach(async () => {
+    const module: TestingModule = await Test.createTestingModule({
+      providers: [
+        TodosService,
+        {
+          provide: TodosDatabaseService,
+          useValue: createMock<TodosDatabaseService>(),
+        },
+      ],
+    }).compile()
+
+    todosService = module.get<TodosService>(TodosService)
+    todosDatabaseService =
+      module.get<TodosDatabaseService>(TodosDatabaseService)
+  })
+
+  it('should call create method with correct arguments', async () => {
+    const todoItem: Omit<Todo, '_id'> = {
+      title: 'Test title',
+      description: 'Test description',
+    }
+    await todosService.create(todoItem as Todo)
```

Comment:
I think we have wrong types here. I assume todosService.create should accept CreateTodoDto as the single argument; and CreateTodoDto should not have _id prop. With these changes no need to do any magic with types